nmsg/msgmod/message.c

```diff
@@ -504,6 +504,20 @@ _nmsg_message_to_json(nmsg_output_t output, nmsg_message_t msg, struct nmsg_strb
 	}
 }
 
+nmsg_res
+_nmsg_message_get_field_value_as_json(nmsg_message_t msg, const char *name, struct nmsg_strbuf *sb) {
+	if (msg->mod == NULL)
+		return (nmsg_res_failure);
+	switch (msg->mod->plugin->type) {
+		case nmsg_msgmod_type_transparent:
+			return _nmsg_message_payload_get_field_value_as_json(msg, name, sb);
+		case nmsg_msgmod_type_opaque:
```

Comment:
is not handling opaque completely ok?  or is it for future work?  add a comment about this

Reply:
This piece of code is copy/paste from _nmsg_message_to_json. What kind of comment should I put ?

Reply:
Dropping the issue for now.